docs/core/diagnostics/built-in-metrics-system-net.md

```diff
@@ -0,0 +1,139 @@
+  * [Instrument: `http.client.request.duration`](#instrument-httpclientrequestduration)
+  * [Instrument: `http.client.request.time_in_queue`](#instrument-httpclientrequesttime_in_queue)
+  * [Instrument: `http.client.active_requests`](#instrument-httpclientactive_requests)
+
+## Meter: `System.Net.NameResolution`
+
+### Instrument: `dns.lookup.duration`
+
+| Name     | Instrument Type | Unit | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dns.lookup.duration` | Histogram | `s` | Measures the time taken to perform a DNS lookup. |
+
+| Attribute  | Type | Description  | Examples  | Presence |
+|---|---|---|---|---|
+| `dns.question.name` | string | The name being queried. | `www.example.com`; `dot.net` | Always |
+
+This metric measures the time take to make DNS requests. These requests can occur by calling
+<xref:System.Net.Dns.GetHostName> or indirectly wihtin higher level APIs on types such as <xref:System.Net.Http.HttpClient>.
+
+Available starting in: .NET 8
+
+## Meter: `System.Net.Http`
+
+### Instrument: `http.client.open_connections`
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `http.client.open_connections` | UpDownCounter | `{connection}` | Number of outbound HTTP connections that are currently active or idle on the client |
+
+| Attribute  | Type | Description  | Examples  | Presence |
+|---|---|---|---|---|
+| `http.connection.state` | string | State of HTTP connection in the HTTP connection pool. | `active`; `idle` | Always |
+| `network.protocol.version` | string | Version of the application layer protocol used. | `1.1`; `2` | Always |
+| `server.address` | string | Host identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. | `example.com` | Always |
+| `server.port` | int | Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. | `80`; `8080`; `443` | If not default (`80` for `http` scheme, `443` for `https`) |
+| `server.socket.address` | string | Server address of the socket connection - IP address or Unix domain socket name. | `10.5.3.2` | Always |
+| `url.scheme` | string | The [URI scheme](https://www.rfc-editor.org/rfc/rfc3986#section-3.1) component identifying the used protocol. | `http`; `https`; `ftp` | Always |
+
+<xref:System.Net.Http.HttpClient>, when configured to use the default <xref:System.Net.Http.SocketsHttpHandler>, maintains a cached pool of network connections for sending HTTP messages. This metric counts how many connections are currently
+in the pool. Active connections are handling active requests. Active connects could be transmitting data or awaiting the client or server. Idle connections aren't handling any
+requests, but are left open so that future requests can be handled more quickly.
+
+Available starting in: .NET 8
```

Comment:
How will we restructure these docs in case new attributes will be introduced in .NET 9? New columns?